components/automate-ui/e2e/admin.e2e-spec.ts

```diff
@@ -103,485 +102,6 @@ describe('Admin pages', () => {
     });
   });
 
-
-  describe('IAM v2', () => {
```

Comment:
Why delete all the token tests here? Or have they just been moved to the new 06_token_mgmt_spec ? Though, seems like the set of tests for token details has been reduced a lot.

Reply:
They've all been moved, short of checking for the heading (API Tokens) and the date. 

Protractor has been on our 💥 list for awhile, so we took this opportunity to port to cypress. You think those extra assertions are worth it? (Would just be some awkwardness around date conversion but 🤷‍♂ )

Reply:
Did not look deeply; if you've considered it already, that's sufficent.